src/libraries/Native/Unix/System.Globalization.Native/pal_localeNumberData.c

```diff
@@ -220,6 +215,7 @@ static int GetCurrencyNegativePattern(const char* locale)
     static const char* Patterns[] = {"(Cn)",
                                      "-Cn",
                                      "C-n",
+                                     "C- n",
```

Comment:
Is this going to shift indices of the remaining patterns, and in turn change the value returned by https://docs.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo.currencynegativepattern?view=netframework-4.8 ?

Reply:
You are right. missed fixing the managed side too and should insert the value at the end of the table.